pkg/ctl/cmdutils/configfile.go

```diff
@@ -192,7 +192,7 @@ func NewCreateClusterLoader(cmd *Cmd, ngFilter *NodeGroupFilter, ng *api.NodeGro
 			*l.ClusterConfig.VPC.NAT.Gateway = api.ClusterSingleNAT
 		}
 
-		api.SetClusterEndpointAccessDefaults(l.ClusterConfig.VPC.ClusterEndpoints)
+		api.SetClusterEndpointAccessDefaults(l.ClusterConfig.VPC)
```

Comment:
Thanks for the quick response and PR. Just tested this, it works perfectly, and unit test is superb 💯 🎉 🥇 

Just a little bit curious why we didn't follow the same approach like other VPC components (e.g. NAT), something like below
```
		if l.ClusterConfig.VPC.ClusterEndpoints == nil {
			l.ClusterConfig.VPC.ClusterEndpoints = api.ClusterEndpointAccess()
		}
```

Additionally, to avoid a same problem, anyone calling `SetClusterEndpointAccessDefaults` must make sure that VPC struct is not nil.

Reply:
In this case I want to be able to set default values individually for each of the fields: private and public

Yes, you have a good point. The caller has to be aware of that.

Reply:
> In this case I want to be able to set default values individually for each of the fields: private and public

:100: